src/main/resources/templates/fragments/navbar.html

```diff
@@ -0,0 +1,85 @@
+<div th:fragment="navbar" id="navbar">
+    <nav class="navbar">
+        <div class="navbar-brand">
+            <a class="navbar-item" href="/">OurShop</a>
+        </div>
+
+        <!-- Navbar Links -->
```

Comment:
As an option to consider, you could also add the navigation links dynamically

Reply:
ok, this one was a bit tricky, but the idea is nice. I created a new model and a new controller class and now the links can be created dynamically